conf/messages

```diff
@@ -224,7 +224,7 @@ ssttp.affordability.set-up-plan-with-adviser.h1 = Set up a payment plan with an
 ssttp.affordability.set-up-plan-with-adviser.p1 = The payment plan you have selected needs to be set up with an adviser.
 ssttp.affordability.set-up-plan-with-adviser.p2 = You can go back to  <a class="govuk-link" id="back-to-income-and-spending" href="{0}">check your income and spending</a>.
 ssttp.affordability.set-up-plan-with-adviser-section.h2 = Connect to an adviser
-ssttp.affordability.set-up-plan-with-adviser-section.p1 = <a class="govuk-link" href="{0}">Use our webchat</a> to set up a payment plan with an adviser online.
+ssttp.affordability.set-up-plan-with-adviser-section.p1 = <a class="govuk-link" href="{0}" rel="noreferrer noopener" target="_blank">Use our webchat</a> to set up a payment plan with an adviser online.
```

Comment:
don't we need something like `(opens in a new tab)` here?

Reply:
Yes I think you're right. It was discussed, but it's not in the ticket. I'll chase.